### PR TITLE
Origin/189 feature train object detection model stage 1

### PR DESCRIPTION
**Pull request by kdigava**

### CODE REVIEW

File: model_training/yolo_runs/train2/results.csv
Comment (hbtasdem) [inline model_training/yolo_runs/train2/results.csv:1]: mAP50 results look solid!!

**Review by hbtasdem — COMMENTED** [review]
No signs of overfitting, train/val loss makes sense and precision is very good. Approving!!

**Review by hbtasdem — APPROVED** [review]
